ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -268,7 +281,7 @@ export const WorkspaceEdit = fp.flow(withCurrentWorkspace(), withCdrVersions(),
       super(props);
       this.state = {
         billingAccounts: [],
-        cdrVersionItems: this.createInitialCdrVersionsList(),
+        cdrVersions: props.workspace ? this.getCdrVersions(props.workspace.accessTierShortName) : this.getCdrVersions(DEFAULT_ACCESS_TIER),
```

Comment:
if props.workspace exists (Edit or Duplicate) then we need to use the existing workspace's access tier.  Otherwise we default to Registered.